lib/plugin.js

```diff
@@ -1,20 +1,38 @@
+<% if (options.methods) { %>
+import { <%= options.methods.join(', ') %> } from 'date-fns'
+<% } else { %>
 import * as dateFns from 'date-fns'
+<% } %>
 
 <% if (options.locales) { %>
 import { <%= options.locales.join(', ') %> } from 'date-fns/locale'
 <% } %>
 
 export default (ctx, inject) => {
+  <% if (options.methods) { %>
+  const dateFns = { <%= options.methods.join(', ') %> }
+  <% } %>
   const dateObj = {...dateFns}
```

Comment:
I think need `else` here

Reply:
dateFns is already defined if options.methods is null because of "import * as dateFns from 'date-fns'" so no else is needed

Reply:
You're right